src/components/tabs/tab.test.tsx

```diff
@@ -27,13 +26,13 @@ describe('EuiTab', () => {
   describe('Props', () => {
     describe('onClick', () => {
       test('is called when the button is clicked', () => {
-        const onClickHandler = sinon.stub();
+        const onClickHandler = jest.fn();
```

Comment:
When converting a component to TS, also convert the related test file.
We don't use the `sinon` library much, so I just replaced it with a parallel library that we do use.